doc/user/content/sql/functions/date_part.md

```diff
@@ -0,0 +1,75 @@
+---
+title: "date_part Function"
+description: "Returns a specified time component from a time-based value"
+menu:
+  main:
+    parent: 'sql-functions'
+---
+
+`date_part` is modeled on the traditional Ingres equivalent to the SQL-standard 
+function [`EXTRACT`](../extract). For PostgreSQL compatibility, `date_part` returns values of type
+[`float`](../../types/float). This can result in a loss of precision in certain uses. Using 
+[`EXTRACT`](../extract) is recommended instead.
+
+## Signatures
+
+TODO: this svg doesn't exist
```

Comment:
I'm not really sure how to generate these images, so any help there is appreciated. An alternative would be to just get rid of this page and add a note to the bottom of the `EXTRACT` docs.

Reply:
https://github.com/MaterializeInc/materialize/tree/main/doc/user#railroad-diagrams-for-sql-grammar

Reply:
Thanks, this should be fixed now